config/daemonconfig.go

```diff
@@ -161,7 +161,7 @@ func NewDaemonConfig(fsDriver string, cfg DaemonConfig, imageID, snapshotID stri
 			backendConfig = &cfg.Config.BackendConfig
 			fscacheID := erofs.FscacheID(snapshotID)
 			cfg.ID = fscacheID
-			cfg.DomainID = fscacheID
+			cfg.DomainID = "global-domain"
```

Comment:
Is domain ID specific to kernel/erofs? In other words, is it a predefined keyword in the Kernel?

Reply:
Yup, @domain_id is a mount option of EROFS.

Reply:
Actually, I mean "global-domain" :-)

Reply:
'global-domain' is not a keyword in the kernel. This is just a name(id) specified by us. In future work, 'domain_id' will be specified by the user.